flocker/provision/_install.py

```diff
@@ -43,6 +43,103 @@
 }
 
 
+def task_client_smoke_test():
+    """
+    Check that the CLI is working.
+    """
+    return run_from_args(['flocker-deploy', '--version'])
+
+
+def task_install_cli(distribution, package_source=PackageSource()):
+    """
+    Install flocker CLI on a distribution.
+
+    :param bytes distribution: The distribution the node is running.
```

Comment:
If you have a dispatcher you can refer to it, otherwise one has to read the code and not just the docstring to figure out what counts as a "distribution".